Zend/zend_execute.h

```diff
@@ -377,7 +377,9 @@ ZEND_API void ZEND_FASTCALL zend_free_extra_named_params(zend_array *extra_named
 
 /* services */
 ZEND_API const char *get_active_class_name(const char **space);
+ZEND_API const char *get_prev_active_class_name(const char **space);
 ZEND_API const char *get_active_function_name(void);
+ZEND_API const char *get_prev_active_function_name(void);
```

Comment:
I wouldn't like to add these new function to public API.
I won't object of you move their implementation into ``zend_dtrace.c``.